src/javascript/js-webshim/minified/polyfiller.init.js

```diff
@@ -0,0 +1,6 @@
+(function($) {
+    $.webshim.bugs.bustedValidity = !('minLength' in document.createElement('input'));
+    $.webshims.setOptions('basePath', Zikula.Config.baseURL+'javascript/js-webshim/minified/shims/'); 
+    $.webshims.activeLang(Zikula.Config.lang);
+    $.webshims.polyfill(Zikula.Config.polyfillFeatures);
+})(jQuery);
```

Comment:
I see now. I assume you are loading the minified versions in `prod`? non-minified in `dev`?